packages/lesswrong/lib/filterSettings.ts

```diff
@@ -92,6 +93,7 @@ export const useFilterSettings = () => {
     collectionName: "Tags",
     fragmentName: "TagBasicInfo",
     limit: 100,
+    skip: !hasSuggestedFilterTags,
```

Comment:
This isn't actually necessary for this PR, but while I was investing how frontpage filter settings worked I realised that we're making this graphql request in a few places, even though it's only need for a LW-only feature so we may as well skip it.